app/views/course/assessment/question/multiple_responses/_form.html.slim

```diff
@@ -1,9 +1,7 @@
 = simple_form_for [current_course, @assessment, @multiple_response_question] do |f|
   = f.error_notification
   = render partial: 'course/assessment/questions/form', locals: { f: f }
-  = f.input :grading_scheme,
-            as: :select,
-            collection: Course::Assessment::Question::MultipleResponse.grading_schemes.keys
+  = f.hidden_field :multiple_choice
```

Comment:
Make this `grading_scheme` and set a default value ? so you don't need to do the type cast yourself.